rancid-tomatillos/cypress/e2e/card_container.cy.js

```diff
@@ -1,23 +1,55 @@
+{/* <reference types="cypress" />  */}
+
+
 describe('CardContainer Test Suite', () => {
 
     beforeEach(() => {
       cy.visit('http://localhost:3000')
     })
 
-  // it("should be able to see all movies, each with their poster, title and rating", () => {
-  //   cy.get(".card").should("exist")
-  // });
+  // it("should display an error message for a 500 status code", () => {
```

Comment:
We will do some research on this so we can implement it properly in the next iteration.